packages/ui/src/views/repo/repo-list/types.ts

```diff
@@ -22,4 +24,5 @@ export interface RepoListProps {
   errorMessage?: string
   searchQuery?: string | null
   setSearchQuery: (query: string | null) => void
+  breadcrumbs: ReactNode
```

Comment:
hm.. i dont like passing Breadcrumbs to pages. ideally breadcrumbs should flow upwards through the tree. will need to discuss this.

Reply:
Removed Breadcrumbs from views